mygraph.py

```diff
@@ -196,8 +194,28 @@ def incident(self, vertex: Vertex) -> bool:
         return self.head == vertex or self.tail == vertex
 
 
-class Graph(object):
-    def __init__(self, directed: bool, n: int=0, simple: bool=False):
+class Graph(Hashable):
+    _generated_label_values = set()
```

Comment:
👍

Reply:
Hiervoor hadden Dorien en Claudia ook al iets van een kleurgenerator gemaak ergens. Dus dit zal ik niet meenemen denk ik. Of we moeten het op deze manier willen oplossen.